<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RandomMultiplierDialog</class>
 <widget class="QDialog" name="RandomMultiplierDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>random multiplier</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>
   <property name="spacing">
    <number>16</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>setup random multiplier</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>34</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>number of copies</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="numberEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="minAngVelEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxAngVelEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>150</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxVelXEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="minVelXEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="minVelYEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxVelYEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="randomizeVelXcheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>randomize velocity x (min-max)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="randomizeVelYcheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>randomize velocity y (min-max)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="randomizeAngVelCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>331</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>randomize angular velocity (min-max)</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="randomizeAngleCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>randomize angle (min-max)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="minAngleEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="maxAngleEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>60</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RandomMultiplierDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeVelXcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minVelXEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeVelXcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxVelXEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeVelYcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minVelYEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeVelYcheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxVelYEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeAngVelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minAngVelEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeAngVelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxAngVelEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeAngleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minAngleEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>randomizeAngleCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxAngleEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
